beacon-chain/powchain/engine-api-client/v1/mocks/client.go

```diff
@@ -15,6 +16,9 @@ type EngineClient struct {
 	ExecutionPayload      *pb.ExecutionPayload
 	Err                   error
 	ExecutionBlock        *pb.ExecutionBlock
+	ErrLatestExecBlock    error
+	ErrExecBlockByHash    error
+	BlockByHashMap        map[[32]byte]*pb.ExecutionBlock
```

Comment:
How big can this get? Are we better using an LRU here?

Reply:
This is just the mock engine client. We are not using it for production : )

Reply:
uh LOL totally missed that. It looked legit